### DISCUSSION TITLE
cmdline switch to autostart viewer

### DISCUSSION THREAD

**Discussion by eadmaster** [doublecmd/doublecmd#1264] [Suggestions]
Can you add a [standalone lister exe](https://www.ghisler.com/lister/), or alternatively a [cmdline switch](https://doublecmd.github.io/doc/en/commandline.html) to start another instance of doublecmd in this mode?

e.g.
````
doublecmd.exe --lister myfile.ext
````

**Comment by JonnyTech** [comment]
Haha, "lister.exe" is a Total Commander application. This same request stems from there. Why to you want the internal viewer as a separate application? It is very basic and there are many better standalone viewers available.

**Comment by Skif-off** [comment]
> Same program as in version **6.5** of Total Commander.

Very necessary program and is kept up to date :) Total Commander has been using "/S=L" for a long time (7.50 Beta 3, 2008), it looks logical and has full functionality.

I join **JonnyTech**'s question: for what purposes?

**Comment by eadmaster** [comment]
i've found TC Lister is pretty fast at viewing big files, even binary files in hex mode.

Also, when combined with [lister plugins](https://totalcmd.net/directory/lister.html) can be used to preview a lot of file formats without the need of installing 3rd party apps.
So on Windows i've made an Explorer shell extension for TC Lister associated to any filetype.

**Comment by eadmaster** [comment]
similar usage to autostart in diffing mode:

`doublecmd --diff myfile.ext myfile2.ext`
